### PR TITLE
Clarify discontinued weekly facts and figures in blog

### PR DESCRIPTION
**Pull request by larswmh**
This PR solves #1994 

DE:
https://www.coronawarn.app/de/blog/2021-10-01-facts-and-figures/
image
https://www.coronawarn.app/de/blog/2021-10-04-key-figures-dashboard/
image

EN:
https://www.coronawarn.app/en/blog/2021-10-01-facts-and-figures/
image
https://www.coronawarn.app/en/blog/2021-10-04-key-figures-dashboard/
image

### CODE REVIEW

File: blog/2021-10-01-facts-and-figures/index_de.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ author: Janina Hoerdt
 layout: blog
 ---
 
-In diesem Beitrag finden Sie in regelmäßigen Abständen [aktualisierte Zahlen und Fakten](/assets/documents/2021-09-30-cwa-daten-fakten.pdf) zur Entwicklung der Corona-Warn-App, wie auch zur Anzahl übermittelter und geteilter Testergebnisse. 
+In diesem Beitrag finden Sie vorherige Zahlen und Fakten zur Entwicklung der Corona-Warn-App, wie auch zur Anzahl übermittelter und geteilter Testergebnisse.
+
+An die Stelle des bis zum 30. September 2021 wöchentlich erschiehenen Übersichtsdokuments trat ein öffentliches, interaktives Dashboard mit den wichtigsten Kennzahlen, welche täglich aktualisiert werden. Weitere Informationen zum Dashboard sind in folgendem Blog zu finden: [Kennzahlen-Dashboard zur Corona-Warn-App](../2021-10-04-key-figures-dashboard/)
```
Comment (MikeMcC399) [inline blog/2021-10-01-facts-and-figures/index_de.md:12]: erschiehenen => erschienenen
```suggestion
An die Stelle des bis zum 30. September 2021 wöchentlich erschienenen Übersichtsdokuments trat ein öffentliches, interaktives Dashboard mit den wichtigsten Kennzahlen, welche täglich aktualisiert werden. Weitere Informationen zum Dashboard sind in folgendem Blog zu finden: [Kennzahlen-Dashboard zur Corona-Warn-App](../2021-10-04-key-figures-dashboard/)
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
One typo but also the link to the other blog doesn't work.
You need to replace with /en/blog etc.

See my comments in https://github.com/corona-warn-app/cwa-website/blob/master/docs/WRITE_BLOG_POSTS.md#link-to-other-locations

Note: Relative paths using ../ (go up one level) are not recommended in blog posts, since they are error prone in this situation. Blog sources are processed and published at two different levels: the blog overview and blog individual detail level.

File: blog/2021-10-01-facts-and-figures/index_de.md (hunk 1 above)
Comment (larswmh) [reply blog/2021-10-01-facts-and-figures/index_de.md]: Corrected it in the latest commit, thanks!

**Comment by MikeMcC399** [comment]
@larswmh 

The links look fine now! You can use relative links in blogs like `../`, but you have to be careful about whether you use them in the blog overview (then go up one level) or in the blog text (go up two levels). It is safer and more readable to do it with `/en/blog` and `/de/blog`. The only disadvantage is that the links are then language dependent.

---
Could you correct another typo? It's not in what you changed. It was there before.

In `/de/blog/2021-10-04-key-figures-dashboard/` i.e. in `index_de.md`

"Warnungen mit grünem und rotem Risiko: Daten aus der Privacy Preserving Analytics, PPA)."

it should be "Privacy Preserving Analytics (PPA)"

There is a comma instead of an opening bracket.

**Review by MikeMcC399 — APPROVED** [review]
LGTM!
Thanks for addressing this issue (and my comments).

**Comment by MikeMcC399** [comment]
@larswmh

This PR seems to have been superseded by https://github.com/corona-warn-app/cwa-website/pull/2059 and this other PR has a typo in it.  Are you working together?

**Comment by dsarkar** [comment]
@MikeMcC399 @larswmh ;-) Yes, we work together, however, probably I did not communicate clear enough that apart from the text changes, that we had already prepared a PR.

**Comment by MikeMcC399** [comment]
https://github.com/corona-warn-app/cwa-website/pull/2062 corrected the typo I mentioned.

**Comment by MikeMcC399** [comment]
@dsarkar 
> Yes, we work together, however, probably I did not communicate clear enough that apart from the text changes, that we had already prepared a PR.

Thanks for your response. I was unsure what had happened.

I compared this PR with the changed published content the main points have been dealt with, although the wording is slightly different.

One point outstanding is the typo correction

from:
- Warnungen mit grünem und rotem Risiko: Daten aus der Privacy Preserving Analytics, PPA).

to:
- Warnungen mit grünem und rotem Risiko: Daten aus der Privacy Preserving Analytics (PPA).

which was included in this PR.

@larswmh 
Would you submit a new PR to correct this? Let me know if I should do it.

**Comment by MikeMcC399** [comment]
Otherwise I think that this PR can be closed.

**Comment by dsarkar** [comment]
@MikeMcC399 If you provide a PR I will merge it. Many thanks!

**Pull request closed without merging**